### ISSUE TITLE
Regarding the RETOUCH dataset

### ISSUE THREAD

**Issue by DBook111** [Zyun-Y/DconnNet#27]
The MICCAI 2017 RETOUCH retinal fluid dataset has three parts, with each part containing 8 volumes. How is the training set and validation set determined for this dataset?

![image](https://github.com/Zyun-Y/DconnNet/assets/101385916/25718034-5745-4875-9730-f03b6aa01272)

![image](https://github.com/Zyun-Y/DconnNet/assets/101385916/11e56733-8f3b-488c-a811-0c4d27ad5351)

![image](https://github.com/Zyun-Y/DconnNet/assets/101385916/f73576d3-8c77-492f-a66c-2b1430b450a9)

**Comment by dreamhonme** [comment]
Excuse me, did you solve it
